### ISSUE TITLE
Going through assignments

### ISSUE THREAD

**Issue by chiefcll** [LarsDenBakker/lit-html-workshop#1]
With the current structure, going through assignments is difficult. I'd recommend changing it to the following setup:

1. Have an index.html + solution.js file that participants work from. The index.html loads solution.js and participants will only use solution.js. Also working in a JS file allows the lit-html vs code plugin to work.
2. The assignments can become Markdown files with instructions. There is a lot of noise when opening assignments, and its hard to load separate assignments in the browser. 
3. Have the main index page include links to LitHTML, LitElement Assignments + Final Solution

**Comment by LarsDenBakker** [comment]
I agree all the separate index.html's are cumbersome to work with, and the lack of highlighting is definitely a concern (thought it's really a bug in the vscode plugin).

One thing I do want to ensure is that people can still copy paste the samples / scaffolding into an online code editor, which is important aspect of our workshop when we do them in a group in real life.

I don't have time right now to make any changes, but I'd be happy to review a PR :)

**Comment by chiefcll** [comment]
NP - I wanted to make sure it would be beneficial before starting the work. I'll open a PR at some point.
